src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -107,6 +109,9 @@ public TableHandler(ClusterManager mgr, String name, TableOptions options)
 
     inQuerying_ = new AtomicBoolean(false);
     lastQueryTime_ = 0;
+
+    this.interceptorManger = new InterceptorManger();
+    interceptorManger.add(new BackupRequestInterceptor(options.enableBackupRequest()));
```

Comment:
I think you can make the line of `new BackupRequestInterceptor(options.enableBackupRequest()` into `InterceptorManager`,  making `BackupRequestInterceptor` invisable to `TableHandler`